### ISSUE TITLE
500 Error When Pulling List Items

### ISSUE THREAD

**Issue by benice25** [pnp/sp-dev-fx-webparts#2052]
### Sample

react-accordion-dynamic-section

### Author(s)

@mikezimm @jyasir @AriGunawan

### What happened?

The dynamic accordion webpart should be pulling in the list items from the selected list. I'm able to see the correct choice of lists, correct categories, correct columns, everything. I configure it accordingly, then apply. But when refreshing the page no results show up. According to the console, there is a 500 error when trying to get the list of items, see errors below.

```
Failed to get list items!
react-accordion-web-part_37a522f90f94b7e2e8bca2f8689bc1a4.js:1 Error: Error making HttpClient request in queryable [500]  ::> {"odata.error":{"code":"-2130575340, Microsoft.SharePoint.SPException","message":{"lang":"en-US","value":"One or more field types are not installed properly. Go to the list settings page to delete these fields."}}}
    at new t (react-accordion-web-part_37a522f90f94b7e2e8bca2f8689bc1a4.js:1)
    at Function.<anonymous> (react-accordion-web-part_37a522f90f94b7e2e8bca2f8689bc1a4.js:1)
    at react-accordion-web-part_37a522f90f94b7e2e8bca2f8689bc1a4.js:1
    at Object.next (react-accordion-web-part_37a522f90f94b7e2e8bca2f8689bc1a4.js:1)
    at s (react-accordion-web-part_37a522f90f94b7e2e8bca2f8689bc1a4.js:1)
```

### Steps to reproduce

1. Pull down repo.
2. NPM-I + Gulp Commands to create a package (_I tested this in DEV for quite some time but moving to PROD is when we get the error_)
3. Install the package and deploy to SP site.
4. Create a list of FAQs
5. Put the Dynamic Accordion webpart on the page.
6. Configure it to pull in the list, category, set title & answer.
7. Save the page and refresh (_no list items are pulled in_)

### Expected behavior

I would expect the gadget to retrieve the list items associated to the list that is selected when configuring the dynamic accordion. 

### Target SharePoint environment

SharePoint Online

### Developer environment

Windows

### Browsers

- [ ] Internet Explorer
- [ ] Microsoft Edge
- [X] Google Chrome
- [ ] FireFox
- [ ] Safari
- [ ] mobile (iOS/iPadOS)
- [ ] mobile (Android)
- [ ] not applicable
- [ ] other (enter in the "Additional environment details" area below)

### Node.js version

v10.24.1

**Comment by AriGunawan** [comment]
That's weird. I have exactly same columns with you but don't get the error.
Maybe there is something wrong on the query for getting the items?
Could you please check on the browser developer tools > network tab > find "getItems" request > payload?

Below is my result
![image](https://user-images.githubusercontent.com/8691985/143723197-21654c6a-77e6-4dae-831c-e16fdacfef0c.png)

**Comment by tospens** [comment]
@benice25 I experienced the same error. The cause was renaming columns. Which made me suspect the getItem query was using the column DisplayName, not the InternalName.

My error was resolved by changing the code in line 119 in the ReactAccordionWebpart.ts file from `key: column.Title` to `key:column.InternalName`

Below is the referenced code:
![image](https://user-images.githubusercontent.com/10114382/151969426-11af2174-353e-4405-a102-7186bd2ccb47.png)
